src/modify_resources.py

```diff
@@ -127,6 +127,25 @@ def reset_cpu_quota(ssh_client, container_id):
     print update_command
     ssh_exec(ssh_client, update_command)
 
+
+# Pins the selected cores to the container (will reset any CPU quotas)
+# Hardcoded cpuset-mems for now
+def set_cpu_cores(ssh_client, container_id, cores):
+    reset_cpu_quota(ssh_client, container_id) # CPU quotas conflict with core pinning
+    rst_cores_cmd = 'docker update --cpuset-cpus={} --cpuset-mems=0 {}'.format(cores, container_id)
+    ssh_exec(ssh_client, rst_cores_cmd)
+    print 'Cores {} pinned to container {}'.format(cores, container_id)
+
+
+# Resetting pinned cpu_cores (container will have access to all cores)
+def reset_cpu_cores(ssh_client, container_id):
+    cores = get_num_cores(ssh_client) - 1
+    core_cmd = '0-{}'.format(cores)
```

Comment:
It turns out that --cpuset-cpus does not take -1 as a value. I can't find any other way to reset this either.